xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/macros.vm

```diff
@@ -2423,6 +2423,11 @@ $string##
 #macro (userPicker_input $multiSelect $parameters)
   #if ($multiSelect)
     #set ($discard = $parameters.put('multiple', 'multiple'))
+    #set ($discard = $parameters.putIfAbsent('title',
+      "$services.localization.render('core.widgets.suggest.userPicker.multi.title')"))
```

Comment:
```suggestion
      $services.localization.render('core.widgets.suggest.userPicker.multi.title')))
```

Same below.

Reply:
Fixed in https://github.com/xwiki/xwiki-platform/pull/2221/commits/200a0e6772b6bf275389cfccc2c1ba0158815fb5